mix.exs

```diff
@@ -65,7 +67,8 @@ defmodule ElixirBench.Mixfile do
     [
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      "test": ["ecto.create --quiet", "ecto.migrate", "test"]
+      "test": ["ecto.drop --quiet", "ecto.create --quiet", "ecto.migrate", "test"],
+      "smoke_test": ["test"],
```

Comment:
I don't think we need to drop our db every time we run the tests :) Similarly, `smoke_test` as an alias for `test` I don't see the win :)

Reply:
I will revert this, doesn't make sense at all.